apps/heft/src/configuration/HeftConfiguration.ts

```diff
@@ -161,13 +180,22 @@ export class HeftConfiguration {
 
   private constructor() {}
 
+  /**
+   * Performs the search for rig.json and initializes the `HeftConfiguration.rigConfig` object.
+   */
+  public async checkForRigAsync(): Promise<void> {
+    if (!this._rigConfig) {
+      this._rigConfig = await RigConfig.loadForProjectFolderAsync({ projectFolderPath: this._buildFolder });
+    }
+  }
```

Comment:
It doesn't really make sense for this to be exposed in the public API.

Reply:
Good point.  I'll make it `@internal` for now. Longer term we should probably come up with a better design for initializing RigConfig.